### PR TITLE
Restore i18next-parser functionality

### PR DESCRIPTION
**Pull request by KevinFCormier**
The commit hook and `npm run i18n:frontend` script recently stopped updating strings.

### CODE REVIEW

**Comment by KevinFCormier** [comment]
@fxiang1 and @dtthuynh can you please confirm that the `nodepool -> node pool` changes in this PR are OK? It looked like some spots were missed.

File: frontend/src/routes/Home/Search/Details/YAMLPage.tsx [hunk 1]
```diff
@@ -247,9 +247,10 @@ export function EditorActionBar(props: {
                         <Button
                             variant="secondary"
                             id="download-resource-button"
+                            icon={<DownloadIcon />}
                             onClick={() => downloadYaml(name, resourceYaml)}
                         >
-                            <DownloadIcon /> {t('public~Download')}
```
Comment (KevinFCormier) [inline frontend/src/routes/Home/Search/Details/YAMLPage.tsx:252]: @zlayne Trying to access the `public` translation namespace broke our automatic updating of our translations file.

**Review by fxiang1 — APPROVED** [review]
Yep I noticed those were missing too. So the change looks good. Thanks Kevin!

File: frontend/src/routes/Home/Search/Details/YAMLPage.tsx (hunk 1 above)
Comment (zlayne) [reply frontend/src/routes/Home/Search/Details/YAMLPage.tsx:252]: Thanks for the catch here.

**Review by zlayne — APPROVED** [review]

**Pull request merged**